doc/usage.md

```diff
@@ -55,13 +56,35 @@ services:
     enabled:
       - cron
     disabled: '*'
+    external:
 ```
 
 Sometimes, the supervisor configuration needs information from the
 configuration file, therefore `supervisor.conf` can instead be
 provided as `supervisor.conf.template`, which will be compiled before
 launching. See, e.g., `services/dask`.
 
+External services can also be used to pull a custom microservice from a Github repository.
+
+They can also be added to the `config.yaml` file under the `services.external` key. Each service should have a unique name and a set of parameters that define what specific repository you want to pull. For example:
+
+```
+services:
+  paths:
+    - ./baselayer/services
+    - ./services
+  enabled:
+    - cron
+  disabled: '*'
```

Comment:
Not really needed to show these here.

Reply:
These were already there, I just added the disable key